tests/migrations/test_0043.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+
+@pytest.mark.django_db()
```

Comment:
Oh, I like test writing for migrations! (I'm not sure if I've seen this before?)